### DISCUSSION TITLE
Tabler & color mode

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#943] [Q&A (Help)] [answered]
I'm using theme-tabler and i've a livewire component in my dashboard
I need to set some colors according with theme color mode (dark or light)
What is the correct method to use for my blades?

Thanks for help

**Comment by karandatwani92** [comment]
Hey @blondie63 

There is nothing to do for it; just continue using tabler theme classes. For example:

I created a [counter livewire component](https://livewire.laravel.com/docs/quickstart#create-a-livewire-component) and used `class="btn"`:
Blade: 
```php
<div>
    <h1>{{ $count }}</h1>
    <button class="btn" wire:click="increment">+</button>
    <button class="btn" wire:click="decrement">-</button>
</div>
```
Dashboard:
```
{{-- resources/views/vendor/backpack/theme-tabler/dashboard.blade.php --}}
@extends(backpack_view('blank'))
@section('content')
    <livewire:counter/>
@endsection
```
Result(Notice counter buttons & text color):
![chrome-capture-2024-3-16](https://github.com/Laravel-Backpack/community-forum/assets/8214221/ffe828a3-7b30-469d-b2c3-acca35c09b2b)

**Reply by karandatwani92** [reply, marked as answer]
And if you want to customize the theme colors. Check this [comment](https://github.com/Laravel-Backpack/community-forum/discussions/940#discussioncomment-9128556)

**Reply by blondie63** [reply]
Thanks
